asn1/src/main.rs

```diff
@@ -1,32 +1,39 @@
 use honggfuzz::fuzz;
-use simple_asn1::*;
-use asn1::*;
+//use simple_asn1::*;
```

Comment:
delete this comment as this is not in use anymore

Reply:
Fixed: https://github.com/lambdaclass/fuzzing_examples/pull/17/commits/b0ff795880768b03874cd94706bc19527442a6a3